fvm/fvm_test.go

```diff
@@ -891,20 +891,6 @@ func TestTransactionFeeDeduction(t *testing.T) {
 				require.Equal(t, minimumStorageReservation, balanceAfter)
 			},
 		},
-		{
```

Comment:
Why is this test removed?

Reply:
It was testing the case where the payer had to have minimum account balance + maximum fees on the account or the transaction would fail. Due to the changes in https://github.com/onflow/flow-core-contracts/pull/331 the payer just needs the minimum account balance on the account, which is already covered by other tests.